scripts/local_k8s/unittests/test_cli_log.py

```
@@ -0,0 +1,143 @@
+            and isinstance(decoded[2], bool)
+        )
+        assert is_command_record, (
+            f"fake tool log line {line_number} must decode to "
+            f"[str, list[str], bool], got {type(decoded).__name__}: {decoded!r}"
+        )
+        entries.append(decoded)
+    return entries
+
+
+@pytest.mark.parametrize(
+    ("case",),
```

Ruff (0.16.1)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `str`

(PT006)